src/bootstrap.rs

```diff
@@ -183,3 +185,58 @@ impl BootStrapHandler {
     ;
   }
 }
+
```

Comment:
Perhaps best to make a ```mod test``` and then if you need a crate for test like rand above it can go in the test module.